ui/admin/app/components/credential-stores/credential-store/actions/index.hbs

```diff
@@ -18,6 +18,12 @@
       {{t 'resources.credential.actions.create'}}
     </dd.Interactive>
   {{/if}}
+  {{#if (and (feature-flag 'vault-worker-filter') @model.isVault)}}
```

Comment:
None blocking comment: asking for learning purposes, did we introduce the feature-flag `vault-worker-filter` within this initiative? or it was already present?

Reply:
That flag has been present for a while. It is an enterprise specific capability.

Reply:
@cameronperera 
Gotcha. I wasn't aware and bc I couldn't see it on the feature dropdown I was 🤔 is this new? But all clear now 😄